tests/app/main/test_frameworks.py

```diff
@@ -7,14 +7,11 @@
 
 from app.main.forms.frameworks import ReuseDeclarationForm
 
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import BytesIO as StringIO
 import mock
 import pytest
 from six.moves.urllib.parse import urljoin
 
+from io import BytesIO
```

Comment:
You could tidy up the order of imports for these files please

https://www.python.org/dev/peps/pep-0008/#imports

```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```

Reply:
Why are you complaining when PEP8 isnt? 😛

Reply:
https://github.com/PyCQA/flake8-import-order Don't make me install the flake8 plugin... (although I am getting tempted)

Reply:
Do it! I am bad at remembering to tidy imports...

Reply:
Ha, I thought it would be Ben who would spot it...

I'll take a quick glance properly. Although initially it looks like it requires more config than I expected (I don't think it runs straight out of the box).

Reply:
Don't fret, I've done it and also removed Six from the app while I was at it.